apps/cowswap-frontend/src/modules/erc20Approve/containers/ActiveOrdersWithAffectedPermit/ActiveOrdersWithAffectedPermit.tsx

```diff
@@ -31,24 +34,29 @@ export function ActiveOrdersWithAffectedPermit({ currency, orderId }: ActiveOrde
 
   if (!ordersWithPermit.length || !isPartialApproveSelectedByUser) return null
 
+  const ordersWithPermitLength = ordersWithPermit.length
   const isPlural = ordersWithPermit.length > 1
-  const orderWord = isPlural ? 'orders' : 'order'
+  const orderWord = isPlural ? t`orders` : t`order`
 
   const titleContent = (
-    <>
-      Partial approval may block <span className={'font-bold'}>{ordersWithPermit.length}</span> other {orderWord}.
-    </>
+    <Trans>
+      Partial approval may block <span className={'font-bold'}>{ordersWithPermitLength}</span> other {orderWord}
+    </Trans>
   )
 
+  const areIs = isPlural ? t`are` : t`is`
```

Comment:
In this case, depending on the language the plurals won't be so simple, and the whole sentence can be shortened or maybe extended.

For example, in Portuguese the sentence would be:

Singular:
- There is 1 existing order...
- Existe 1 ordem (alternatively, closer to the source: "Há 1 ordem existente", which is a bit redundant)

Plural:
- There are 2 existing orders...
- Existem 2 ordens (Há 2 ordens existentes)

Any suggestions on how to better handle this?